client/blocks/importer/wordpress/upgrade-plan/test/upgrade-plan-details.tsx

```diff
@@ -0,0 +1,103 @@
+/**
+ * @jest-environment jsdom
+ */
+import { SitePlanPricing } from '@automattic/data-stores';
```

Comment:
Nit: this should be a type import.
```suggestion
import { type SitePlanPricing } from '@automattic/data-stores';
```